tools/probes/wave.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Adrian Bonislawski <adrian.bonislawski@intel.com>
+ */
+
+#ifndef __WAVE_H__
```

Comment:
I hope we do not need to redefine all this wav format related stuff here. Are there any existing headers in packages we already depend on available?

Reply:
@mmaka1 I've seen that every audio processing lib  just redefine it over and over and I'm not aware of any 'standard' simple wav lib + it needs suitable license. If you know any lib feel free to share.

Reply:
Fwiw, ALSA (aplay) also has a definition of this too, seems like it's redefined everywhere.